week-5/C-promises/exercise.js

```diff
@@ -74,40 +94,43 @@ function exercise5() {
 */
 function exercise6() {
   // Write your implementation here
+  var promise6 = Promise.reject("Something went wrong!");
```

Comment:
Like above, there's a longer but more flexible version of doing this. Do you know what it is?

And just to be clear, this isn't wrong, I'm just checking if you know the other ways of doing it 😄